images/rpc.proto

```diff
@@ -60,7 +60,8 @@ enum criu_pre_dump_mode {
 };
 
 message criu_opts {
-	required int32			images_dir_fd	= 1;
+	optional int32			images_dir_fd	= 1;
```

Comment:
is it safe to convert `required` to `optinal`? rpc.proto has to be backward compatible.

Reply:
The old CRIU with a library using newer rpc.proto would fail, but new CRIU with old library would work the same. I've changed it so that older-CRIU scenario will fail only if using the images_dir. (Assuming code does not set both)